src/kbc_modules/offline_sev_kbc/mod.rs

```diff
@@ -26,9 +26,9 @@ pub struct AnnotationPacket {
     // Key ID to manage multiple keys
     pub kid: String,
     // Encrypted key to unwrap
-    pub wrapped_data: Vec<u8>,
+    pub wrapped_data: String,
     // Initialisation vector
-    pub iv: Vec<u8>,
+    pub iv: String,
```

Comment:
Consider adding `(base64-encoded)` to both comments above the fields (if I understand correctly, the `String` is for base64-string).